Expensify/App
# Changed Label on Company and Personal Address Fields

### Details


### Fixed Issues


### Tests / Web QA

1. Head over to http://localhost:8080/bank-account/new or http://staging.new.expensify.com/bank-account/new
2. Click `Connect Manually`
3. Enter `123123123` for both fields
4. Observe the label on the Company Address input.


### Tested On


### Screenshots


#### Web




#### Mobile Web


#### Desktop


#### iOS


#### Android



cc @MitchExpensify
